apidoc/Titanium/UI/iOS/iOS.yml

```diff
@@ -917,6 +917,26 @@ properties:
     since: "5.4.0"
     osver: {ios: {min: "8.0"}}
 
+  - name: BLUR_EFFECT_STYLE_DEFAULT
+    summary: |
+        Use with [BlurView.effect](Titanium.UI.iOS.BlurView.effect) to specify a blur effect.
+    description: |
+        Creates a blurring effect in the view. The blurring effect is the default one specified in iOS 10.
```

Comment:
The blurring effect is the `default` one specified in iOS 10.
vs
The blurring effect is the `regular` one specified in iOS 10.